src/pal/src/file/disk.cpp

```diff
@@ -136,9 +137,11 @@ GetDiskFreeSpaceW(
     }
     else
     {
-        statfsRetVal = statfs( "/", &fsInfoBuffer );
+        dirName = "/";
```

Comment:
The warning was that in case the statfs here failed, we were using uninitialized `dirNameBuffer` variable in `FILEGetProperNotFoundError( dirNameBuffer, &dwLastError );`. Since `dirNameBuffer` is not a `const char *`, I have added the `dirName` that is const so that I can assign a pointer to the literal "/" to it without casting off the constness.